src/DigitizationRunner.cxx

```diff
@@ -544,6 +582,11 @@ void DigitizationRunner::processRootFiles() {
             Float_t z_min_cut = -1;
             Float_t z_max_cut = -1;
             Float_t proj_track_2D_cut = -1;
+
+            int nRedpix;
+            vector<uint16_t>*   redpix_ix = new vector<uint16_t>();
+            vector<uint16_t>*   redpix_iy = new vector<uint16_t>();
+            vector<uint16_t>*   redpix_iz = new vector<uint16_t>();
```

Comment:
This is tricky. In theory there should be no need for pointers. Just vectors should work. However, to put them into the branches with address either you put redpix_ix.data(), or we need the pointer.
I am not fully aware if smart pointer would solve here with branches